extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -23,20 +23,10 @@ public static function register() {
 	public function onGetRailModuleList( Array &$railModuleList ) {
 		global $wgCityId, $wgTitle, $wgUser;
 
-		if ( ( new Helper() )->userCanEditJsPage( $wgTitle, $wgUser ) ) {
-			$pageStatus = \F::app()->sendRequest(
-				'ContentReviewApiController',
-				'getPageStatus',
-				[
-					'wikiId' => $wgCityId,
-					'pageId' => $wgTitle->getArticleID(),
-				],
-				true
-			)->getData();
-
+		if ( $wgUser->isLoggedIn() ) {
```

Comment:
As this is not checking `->isJsPage()` anymore (the check was in `userCanEditJsPage`) the rail module will now show up for all pages, not just JS pages in the MediaWiki namespace